platform-sdk/docs/core/dynamic-address-book/roster/glossary-of-terms.md

```diff
@@ -0,0 +1,100 @@
+# Roster Glossary of Terms
+![Event Window Terminology](./EventWindowTerminology.drawio.svg)
+![Roster Terminology Diagram](./RosterTerminologyDiagram.drawio.svg)
```

Comment:
The box with the question marks needs to be removed.

Reply:
I modified the name of the box to `RosterForRound`